packages/pwa-kit-react-sdk/src/webpack/config.js

```diff
@@ -187,7 +187,16 @@ const common = {
         }),
 
         new CopyPlugin({
-            patterns: [{from: 'app/static/', to: 'static/'}]
+            patterns: [
+                {from: 'app/static/', to: 'static/'},
+                {
+                    from: 'config/',
+                    to: 'config/',
+                    globOptions: {
+                        gitignore: true
```

Comment:
There's no requirement for anyone to use git with our stuff. I think you want to exclude a local config file here, right? Can we exclude that file explicitly, so that this would work across all SCMs, if so?

Reply:
Makes sense. I've changed it to a normal glob pattern